img/t/talktalk.co.uk.svg

```diff
@@ -0,0 +1 @@
-7.087-7.76-7.087-5.025 0-7.197 1.982-7.197 3.922 0 1.765 2.245 1.952 2.893 1.929a.343.343 0 0 0 .245-.154v-.003c.626-1.158 1.803-2.249 4.014-2.249 2.32 0 3.008.702 3.092 2.41a.471.471 0 0 1-.483.437l.003.003c-1.772.159-7.848.85-9.64 2.926-.792.916-.988 1.943-.988 3.205 0 2.638 2.092 4.482 5.53 4.482 1.955 0 3.691-.62 4.91-1.704a.758.758 0 0 1 .168-.117.875.875 0 0 1 .455-.116c.104.003.253.019.357.122a.389.389 0 0 1 .087.118c.194.458.463.8.817 1.037 1.185.797 2.688.418 3.157.241.73-.278.718-.521.59-.979-.183-.631-.258-2.783-.258-4.523zM92.178.487c-.55-.368-1.168-.482-1.722-.484a4.061 4.061 0 0 0-1.486.246c-.708.273-.693.511-.567.958.177.614.258 2.713.264 4.414l.003 1.302v13.554c0 1.651.958 2.085 2.205 2.085h.508c1.213 0 2.09-.51 2.09-2.13l-.05-16.822v-.114c-.013-1.415-.47-2.487-1.246-3.008zM80.934 15.795c0 1.728-1.651 3.318-4.014 3.318-1.587 0-2.272-.62-2.272-1.59 0-1.183 1.846-2.422 5.41-2.657l.2-.01h-.004c.368 0 .668.274.68.619zm4.683.985.008-3.9c0-4.485-1.045-7.087-7.76-7.087-5.026 0-7.199 1.982-7.199 3.922 0 1.765 2.247 1.952 2.896 1.929a.343.343 0 0 0 .245-.154v-.003c.626-1.158 1.803-2.249 4.013-2.249 2.32 0 3.008.702 3.093 2.41a.471.471 0 0 1-.484.437l.003.003c-1.772.159-7.847.85-9.639 2.926-.792.916-.989 1.943-.989 3.205 0 2.638 2.09 4.482 5.53 4.482 1.956 0 3.692-.62 4.91-1.704a.758.758 0 0 1 .169-.117.876.876 0 0 1 .454-.116c.104.003.253.019.357.122a.387.387 0 0 1 .088.118c.194.458.463.8.817 1.037 1.185.797 2.688.418 3.156.241.73-.278.72-.521.588-.979-.18-.631-.257-2.783-.257-4.523zm26.983 3.528c-.533-.394-1.932-2.03-3.016-3.387l-.399-.5-1.604-2.12c-.478-.669-.396-1.36.255-2.069a15.146 15.146 0 0 1 .725-.768l.003-.002.163-.167.056-.058c.158-.159.301-.305.43-.437.73-.714 1.387-1.312 1.718-1.516.393-.243.613-.455.287-.994h-.005c-.003-.003-.003-.005-.006-.011-.169-.315-1.107-1.254-2.463-1.342-.011-.003-.02-.003-.03-.005-.014 0-.026 0-.037-.003h-.051c-.03 0-.061-.003-.092-.003h-.047c-.031.003-.059.003-.09.005-.03.003-.059.003-.09.005a2.032 2.032 0 0 0-.629.154l-.093.04c-.225.102-.438.24-.626.415l-.056.055a12.66 12.66 0 0 0-.68.644l-2.539 2.4c-.306.288-.728.733-1.5.733-.55 0-.935-.373-.935-.945l.009-6.824v-.03c0-.029 0-.055-.004-.085h.004c-.014-1.416-.475-2.487-1.256-3.008C99.449.117 98.831.003 98.275 0a4.086 4.086 0 0 0-1.488.246c-.705.272-.694.51-.564.958.14.487.218 1.91.25 3.326v15.844c0 1.664.882 2.146 2.174 2.146h.511c1.222 0 2.106-.513 2.106-2.146v-.243c.026-.48-.005-3.051-.005-3.451 0-.797.531-.987.908-.987.929 0 1.272.5 1.845 1.154.272.307 2.458 2.885 3.629 4.2h-.003c.941 1.058 1.789 1.471 2.671 1.471 1.331 0 2.281-.879 2.492-1.196.369-.512.17-.742-.201-1.015z"/><path d="M71.054 4.824c1.502-.012 2.643-.444 3.193-1.174.846-1.12.438-2.54.256-2.985a.492.492 0 0 0-.442-.32h-16.34c-1.504.013-2.643.444-3.194 1.174-.845 1.12-.438 2.54-.255 2.985a.49.49 0 0 0 .441.32h6.41a.94.94 0 0 1 .85.672v14.903c0 .793.21 1.315.568 1.643.387.384.958.522 1.629.522h.505c1.213 0 2.09-.511 2.09-2.13V5.761c0-.506.395-.91.89-.937 0 0 3.36 0 3.399-.003z"/></svg>
```

Comment:
I think a single "T" is more readable here, and sufficient as well.
![](https://raw.githubusercontent.com/phallobst/phallobst/main/resterampe/talktalk.co.uk.svg)
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" width="22.22" height="22.22" style="fill:#394347"><path d="M17.78 4.48c1.5-.01 2.64-.45 3.19-1.18.85-1.11.44-2.53.26-2.98a.5.5 0 0 0-.45-.32H4.44c-1.5.01-2.64.44-3.2 1.17C.4 2.3.82 3.71 1 4.16a.49.49 0 0 0 .45.32h6.4a.94.94 0 0 1 .86.67v14.9c0 .8.2 1.32.56 1.65.39.38.96.52 1.63.52h.5c1.22 0 2.1-.51 2.1-2.13V5.42c0-.51.4-.91.89-.94h3.4z"/></svg>
```